tests/expedition/test_ship_config.py

```diff
@@ -0,0 +1,168 @@
+from pathlib import Path
+
+import pytest
+
+from virtualship.expedition.schedule import Schedule
+from virtualship.expedition.ship_config import ConfigError, ShipConfig
+from virtualship.utils import get_example_config, get_example_schedule
+
+expedition_dir = Path("expedition_dir")
+
+
+@pytest.fixture
+def schedule(tmpdir):
+    out_path = tmpdir.join("schedule.yaml")
+
+    with open(out_path, "w") as file:
+        file.write(get_example_schedule())
+
+    schedule = Schedule.from_yaml(out_path)
+
+    return schedule
```

Comment:
I think we can do the same for schedule as the suggestions below with ship_config.